src/Mod/TechDraw/Gui/ViewProviderDimension.cpp

```diff
@@ -264,7 +275,8 @@ double ViewProviderDimension::prefWeight() const
 
 int ViewProviderDimension::prefStandardAndStyle() const
 {
-    return Preferences::getPreferenceGroup("Dimensions")->GetInt("StandardAndStyle", STD_STYLE_ISO_ORIENTED);
+    return static_cast<int>(
+                Preferences::getPreferenceGroup("Dimensions")->GetInt("StandardAndStyle", STD_STYLE_ISO_ORIENTED));

```

Comment:
Static cast int to int?

Reply:
Doesn't GetInt return a long?  lint thinks so.